src/almanac/introduction.md

```diff
@@ -1,3 +1,72 @@
+> _Its guiding philosophy is based on the principle of the ‘garden in motion’: do the most __for__, the minimum __against__."_
+> \- Gilles Clément
+
+The content here is inspired by ecological systems (for example, as in [_planetary gardening_](https://www.architectural-review.com/essays/in-practice/in-practice-gilles-clement-on-the-planetary-garden)).
+We also are galvanized by the [_scientific method_](https://en.wikipedia.org/wiki/Scientific_method), and [_almanacs_](https://en.wikipedia.org/wiki/Almanac) (or earlier [_menologia rustica_](https://en.wikipedia.org/wiki/Menologia_rustica)) in documenting, expecting, and optimizing how we plan for time-based changes in agriculture (among other practices and traditions).
+
+```{figure} assets/almanac-influencing-software.png
+---
+height: 270px
+alt: A book shown as influencing software within the cycle of time.
+---
+_Almanacs help us understand and influence the impacts of time on the things we grow._
+```
+
+Almanacs within the context of computers can be interpreted as meta-software which use objective, shared experience to help encourage growth and avoid unwanted decay.
+The Software Gardening Almanac helps share similar knowledge on cultivation and augmentation in reaction or preparation for change in order to nurture software (and software practitioners) for long periods of time.
+We aspire to define, practice, and continually improve a craft of _software gardening_ to nourish existing or yet to be software projects and embrace and more resilient future.
+
+## Motivation
+
+```{figure} assets/xkcd_dependency.png
+---
+height: 330px
+alt: \'Dependency\' comic by Randall Munroe, XKCD.
+---
+_We depend on a delicate network of software which changes over time.<br>(Image source: ['Dependency' comic by Randall Munroe, XKCD](https://xkcd.com/2347/))_
+```
+
+Our world is surrounded by systems of software which enable and enhance life.
+These systems are both invisible and sometimes brittle, breaking in surprising ways (for example, beneath uneven pressures as in the above XKCD comic).
+At the same time, [loose coupling](https://en.wikipedia.org/wiki/Loose_coupling) of these software systems also forms the basis of innovation through multidimensional, [emergent](https://en.wikipedia.org/wiki/Emergence) growth patterns.
+We seek to embrace these software systems which are innately in motion, embracing diversity without destroying it to perpetuate discovery and enable future life.
+
+> _"Start where you are. Use what you have. Do what you can."_
+> \- Arthur Ashe
+
+We suggest reading or using this content however it best makes sense to a reader.
+Just as with gardening, sometimes the best thing to do is jump in!
+Each section is structured in a modular fashion, providing insights without uncited prerequisites.
+Links and other reference materials for further reading where beneficial (please let us know if we can improve!).
```

Comment:
```suggestion
We provide links and other reference materials for further reading where beneficial.
Please let us know how we can improve!
```

How can they let us know? Do we direct them to the GitHub anywhere or something else alternatively?